react/src/components/molecules/InputRadioGroup/index.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import InputRadio from '../../atoms/forms/InputRadio';
+import ErrorMessage from '../../atoms/forms/ErrorMessage';
+import './style.css';
+
+class InputRadioGroup extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selected: this.props.defaultSelected
+    };
+  }
+
+  handleChange = (selected, value, event) => {
+    if (selected !== this.state.selected) {
+      this.setState({ selected });
+      if (typeof this.props.onChange === 'function') {
+        const { name } = this.props;
+        this.props.onChange({ selected, name, event });
+      }
+    }
+  }
+
+  render() {
+    const itemsClasses = classNames({
+      'ma__input-group__items': true,
+      'ma__input-group__items--inline': this.props.inline,
+      'ma__input-group__items--outline': this.props.outline
+    });
+    const titleClasses = classNames({
+      'ma__input-group__title': true,
+      'ma__input-group__title--error': this.props.error,
+      'ma__input-group__title--disabled': this.props.disabled
+    });
+
+    return(
+      <React.Fragment>
+        <fieldset>
+          <div className="ma__input-group">
+            <legend className={titleClasses}>
+              {this.props.title}
+            </legend>
+            <div className={itemsClasses}>
+              {this.props.radioButtons.map((radioButton, index) => {
+              const isChecked = radioButton.value === this.state.selected;
+              return(
+                <div className={`ma__input-group__item item-${this.props.radioButtons.length}`}>
+                  <InputRadio
+                    {...radioButton}
+                    name={this.props.name}
+                    onChange={this.handleChange}
+                    checked={isChecked}
+                    required={this.props.required}
+                    outline={this.props.outline}
+                    error={this.props.error}
+                    disabled={this.props.disabled}
+                    // eslint-disable-next-line react/no-array-index-key
+                    key={`InputRadioGroup-${radioButton.id}-${index}`}
+                  />
+                </div>
+              );
+            })}
+            </div>
+          </div>
+        </fieldset>
+        {this.props.errorMsg && this.props.error &&
+        <ErrorMessage status={true} error={this.props.errorMsg} inputID={this.props.name} />}
```

Comment:
It's actually okay to drop error, here, as status defaults to error. Also inputID is actually inputId:
```suggestion
        <ErrorMessage error={this.props.errorMsg} inputId={this.props.name} />}
```